qemu/tests/sev_basic_config.py

```diff
@@ -0,0 +1,48 @@
+from avocado.utils import process
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sev basic test on Milan and above host:
+    1. Check host sev capability
+    2. Boot sev VM
+    3. Verify sev enabled in guest
+    4. Check sev qmp cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sev test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    output = process.getoutput(params["sev_host_check"], shell=True)
+    if output not in params["module_status"]:
```

Comment:
Strictly, it's better to use params.objects('module_status') , then we check the keywords only, e.g. 'y', 'Y', if you check the string, it's not that accurate
of course, you need to set module_status = y Y 1 (use blank space instead of ') in cfg

Reply:
Yours is OK for me

Reply:
Thanks zhenchao, update code.